integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -106,7 +106,8 @@ def _get_catalog_content_keys(self, enterprise_customer_catalog):
             integrated_channel_code=self.enterprise_configuration.channel_code(),
             enterprise_customer_catalog_uuid=enterprise_customer_catalog.uuid,
             plugin_configuration_id=self.enterprise_configuration.id,
-            deleted_at__isnull=True,
+            remote_deleted_at__isnull=True,
+            remote_created_at__isnull=False,
```

Comment:
I think I figured this out, with your change since we create the transmission audits before they're sent, if for some reason the transmission isn't sent (errors or chunk size or whatever) we would still hold the record but not have a created. Is this right?